tesseract-server/src/handlers/metadata.rs

```diff
@@ -61,6 +75,62 @@ pub fn members_handler(
 }
 
 
+pub fn get_cube_metadata(
+    mut cube_details: CubeMetadata,
+    ll_config: LogicLayerConfig,
```

Comment:
Can we make this a reference (`&ll_config`) so we don't keep cloning it multiple times?